composer.json

```diff
@@ -18,13 +18,13 @@
         "php": "^7.1.3",
         "ext-json": "*",
         "dragonmantank/cron-expression": "^2.0",
+        "hmazter/laravel-schedule-list": "^1.2",
```

Comment:
This shouldn't be here. As is installed on the client application when he runs the command.

Reply:
But you should add it to require-dev, so you are able to run the tests.

Reply:
Moved thanks